composer.json

```diff
@@ -59,7 +65,7 @@
     },
     "require-dev": {
         "ext-sqlite3": "*",
-        "alt-three/testbench": "^4.0",
+        "alt-three/testbench": "v5.0.0",
```

Comment:
required choice to pin to a specific version?

Reply:
This is our forked version of it with the naming change to match the paths now used in the cachet project.

Reply:
oh ok, so we will be changing with our changes anyway